specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2021-09-01-preview/common/AlertTypes.json

```diff
@@ -39,7 +39,7 @@
       }
     },
     "AttackTactic": {
-      "description": "The severity for alerts created by this alert rule.",
+      "description": "A list of relevant mitre attacks.",
```

Comment:
This type is extensible enum, not a list?

Reply:
This type is an enum. It is not directly related to my changes. I simply use it in my Bookmarks, and I noticed the description here was wrong. Probably a copy&paste error when this file was created.